### DISCUSSION TITLE
Inquiry about Implementing the plugin in Widget Filter

### DISCUSSION THREAD

**Discussion by Morgnus** [malzariey/filament-daterangepicker-filter#70] [Ideas]
@malzariey I hope this message finds you well. First of all, I would like to express my appreciation for your work on this Filament plugin, i find it very useful.

I was wondering if there are any plans to integrate this daterange picker into the Widget filter of Filament in the future?

Such an integration would be a great addition and enhance the filtering capabilities of chart widget. I believe it could benefit many developers who are using or planning to use the date range plugin for their projects.

**Comment by malzariey** [comment]
Let me investigate and get back to you

**Comment by malzariey** [comment]
This will be possible with filamentphp version 4
